public/docs/_examples/intro/dart/lib/logger_service.dart

```diff
@@ -0,0 +1,20 @@
+library developer_guide_intro.logger_service;
+
+import 'dart:html';
+
+import 'package:angular2/angular2.dart';
+
+@Injectable()
+class Logger {
+  void log(dynamic msg) {
+    window.console.log(msg);
+  }
+
+  void error(dynamic msg) {
```

Comment:
It looks like that source code isn't in the page. I suspect the Logger is just an example of a Service. We can leave it as-is, I think.